src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAddAction.java

```diff
@@ -124,7 +124,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.commentId = commentId;
         data.showCommentToString = joinParticipantTypes(createdComment.showCommentTo, ",");
         data.showGiverNameToString = joinParticipantTypes(createdComment.showGiverNameTo, ",");
-        
+
+        data.setCommentIds(commentId);
```

Comment:
Can you explain why this needs to be done? There are a few places where `InstructorFeedbackResponseCommentAjaxPageData` is used, but this is not done in the other places.